styles/styles.css

```diff
@@ -1094,3 +1096,11 @@ main .section.blue a.button.secondary::after {
 .tag-green {
   background-color: var(--green-tag);
 }
+
+main .products-container.section.light-sky-blue h2 {
```

Comment:
Should these better be part of the product block ?

Reply:
@rhudea I wasn't sure since this is outside of the block and only targeting the section containing the block, what do you think should I put it inside of the block?